docs/_static/mxnet-theme/index.html

```diff
@@ -25,16 +21,16 @@
 <div class="section-tout">
   <div class="container">
     <div class="row">
+      <div class="col-lg-4 col-sm-12">
+          <h3>MXNet 1.0 Released</h3>
+          <p>We're excited to announce the release of MXNet 1.0! Check out the release notes for latest updates.</p>
+          <a href="https://github.com/dmlc/mxnet/releases">Learn More</a>
```

Comment:
This link should be: https://github.com/apache/incubator-mxnet/releases/tag/1.0.0

Reply:
Fixed this and pushed the PR.